espressif/vscode-esp-idf-extension
# Bugfix/tarxz install

## Description

This PR add the following bug fixes

- Fix tar.xz install issues in extension by removing tool version from install command.
- Add idf.notificationMode to `vscode.window.withProgress({},()=>{})` functions to show progress in status bar when idf.notificationMode is set to `All` or `Notification`.
- Add ESP32-P4 to extension keywords in package.json
- Update tasks.json for debug adapter task. 

Fixes #1114
Fixes #1113 
Fixes #1063

## Type of change

Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Try to configure extension with `ESP-IDF: Configure ESP-IDF extension` for ESP-IDF v5.2 in Linux. GCC Tools should be installed correctly and building any ESP-IDF v5.2 example Project should work.
2. Execute `ESP-IDF: Build your project` with `idf.notificationMode` set to `All` or `Notification`. The progress notification should be shown. Change `idf.notificationMode` to a different value like `Output`. The progress notification should not be shown.
3. Observe results.

- Expected behaviour: 

For ESP-IDF v5.2 the GCC toolchain should be installed and building example esp-idf project should work

Progress notification should be able to enable and disable based on `idf.notificationMode` configuration setting.

- Expected output:

## How has this been tested?

Manual testing explained above.

**Test Configuration**:
* ESP-IDF Version: 5.2
* OS (Windows,Linux and macOS): Linux

## Checklist
